tools/build-base-images.sh

```diff
@@ -47,7 +47,7 @@ DOCKER_TARGETS="${DOCKER_TARGETS:-${defaultTargets}}"
 # * docker buildx create --name multi-arch --platform linux/amd64,linux/arm64 --use
 # * export DOCKER_ARCHITECTURES="linux/amd64,linux/arm64"
 # Note: if you already have a container builder before running the qemu setup you will need to restart them
-"${ROOT}/tools/docker" --push --no-cache --no-clobber --targets="${DOCKER_TARGETS}"
+#"${ROOT}/tools/docker" --push --no-cache --no-clobber --targets="${DOCKER_TARGETS}"
```

Comment:
Did you mean for this to get commented out?

Reply:
nope